sim/paladin/judgement.go

```diff
@@ -36,13 +36,43 @@ func (paladin *Paladin) registerJudgement() {
 			// rolling on the spell hit table. If it succeeds, the actual Judgement of Command rolls on the
 			// melee special attack crit/hit table, necessitating two discrete spells.
 			// All other judgements are cast directly.
-			if paladin.currentJudgement.SpellCode == SpellCode_PaladinJudgementOfCommand {
-				spell.CalcAndDealOutcome(sim, target, spell.OutcomeMagicHit)
-			} else {
-				paladin.currentJudgement.Cast(sim, target)
+			
+			// Phase 1-3
+			//if paladin.currentJudgement.SpellCode == SpellCode_PaladinJudgementOfCommand {
+			//	spell.CalcAndDealOutcome(sim, target, spell.OutcomeMagicHit)
+			//} else {
+			//	paladin.currentJudgement.Cast(sim, target)
+			//}
+			// paladin.currentSeal.Deactivate(sim)
+			
+			// Phase 4 - (Not tied to T1 6pc bonus) - Judge all Seals (2 possible without 6pc, 2 or 3 with 6pc TBD)	
+			for _, sealAura := range paladin.aurasSoC {
+				if sealAura.IsActive() {
+				        spell.CalcAndDealOutcome(sim, target, spell.OutcomeMagicHit)
+					//paladin.spellsJoC[i].Cast(sim, target)
+					sealAura.Deactivate(sim)
+				}
+			}
+			
+			for i, sealAura := range paladin.aurasSotC {
+				if sealAura.IsActive() {
+					paladin.spellsJotC[i].Cast(sim, target)
```

Comment:
Do we potentially need to do a `nil` check on these for lower levels where some ranks may not be known?

Reply:
I just tried running the sim at level 25 and 40 and saw no issues. I can try reviewing the code to see if it is needed.

Reply:
Though reviewing the code, I don't why it doesn't seem to cause issues. I can add a nil check to each of them.